api/rest/v2/observation.md

```diff
@@ -1,39 +1,43 @@
 ---
 layout: default
 title: Observation
-nav_order: 2
+nav_order: 3
 parent: REST (v2)
 grand_parent: API
 published: true
 permalink: /api/rest/v2/observation
 ---
 
+{:.no_toc}
 # /v2/observation
 
-This API fetches statistical observations. An observation is associated with an
-entity and variable at a particular date. For example, “population of USA in
+* TOC
+{:toc}
+
+The Observation API fetches statistical observations. An observation is associated with an
+entity and variable at a particular date: for example, “population of USA in
 2020”, “GDP of California in 2010”, “predicted temperature of New York in 2050”,
 and so on.
 
-When querying observations, you need to provide variables, entities, and date.
+When querying observations, you need to provide variable, entities, and dates.
 Variables are specified as a list in the form of
 
-```json
+<pre>
```

Comment:
curious if you see better output using pre. my understanding was marking it json helped with highlighting / formatting (e.g. bolding and colors). i noticed that the formatting gets dropped compared to docs.datacommons.org

Reply:
can we keep the formatting, and separately figure out how to update the syntax highlighter to use italics for variables? there must be a way to do this... (i think the readability of the current highlighting helps a lot, even if not in conformance with the style guide). it can be a separate project to apply the style guide.

https://jekyllrb.com/docs/liquid/tags/ (see code snippet highlighting)
https://github.com/datacommonsorg/docsite/blob/master/_sass/friendly_code.scss